eclipse-theia/theia-blueprint
# Structure of download area / Naming of files #51

#### What it does
* change updater to check platform subdirectories of latest
* remove version number from filename

Contributed on behalf of STMicroelectronics

Signed-off-by: Johannes Faltermeier <jfaltermeier@eclipsesource.com>

#### How to test
Build the installers. Files names should not include the version number anymore. 

Updates should work again, which is a bit hard to test. On Unix system there are logs in ~/.config/Theia Blueprint/logs/main.log
(See https://www.npmjs.com/package/electron-log for other OS paths)
Without the fix this should log `[error] Error: Error: Cannot find channel "latest-linux.yml" update info: HttpError: 404 Not Found` after the start. 
With the fix this should become `[info] Update for version 1.11.0 is not available (latest version: 1.11.0, downgrade is disallowed).`

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
